### DISCUSSION TITLE
delete vertex,but can't finish

### DISCUSSION THREAD

**Discussion by codegzx** [JanusGraph/janusgraph#3061] [General]
I use it in atlas to manage metadata, there is a problem when I delete a vertex , the system can't delete all of the associated edges in the other vetex. so my atlas server throw exception that instance with given guid is not exists when  I search the exists vertex to fetch its relations.

**Comment by li-boxuan** [comment]
What you described sounds like half-edge as documented [here](https://docs.janusgraph.org/advanced-topics/eventual-consistency/#temporary-inconsistency). Right now what you could do is manually remove those edges.

**Reply by codegzx** [reply]
thanks.
I had remove those edges. I am going to config the storage.buffer-size and obseravtion it for a period of time.
